xml/depl_require.xml

```diff
@@ -1406,6 +1406,110 @@
     For storage requirements, see <xref linkend="sec.depl.req.storage.hardware.store"/>.
    </para>
   </sect2>
+
+<sect2 xml:id="sec.depl.req.hardware.monnode">
+   <title>&monnode;</title>
+   <para>
+     The Monasca agent roles, such as <literal>monasca-agent</literal> and
+     <literal>monasca-log-agent</literal>, should be assigned to all nodes,
+     regardless of their other roles. However, the
+     <literal>monasca-server</literal> role must be deployed on a dedicated physical machine
+     referred as &monnode;. Hardware required for the &monnode; are as follows:
```

Comment:
Are you sure? The diff still shows the old version ("Hardware required for the &monnode; are as follows").